emulator/accounts_test.go

```diff
@@ -835,19 +837,20 @@ func TestRemoveAccountKey(t *testing.T) {
 	assert.NoError(t, err)
 
 	// key at index 1 should not be revoked
```

Comment:
```suggestion
	// key at indices 1 and 2 should not be revoked
```

Reply:
Updated in https://github.com/onflow/flow-emulator/pull/511/commits/2ee07522ad27fc650a07df3b7748b769493ad517